example/HMI/app/css/colors.module.css

```diff
@@ -0,0 +1,180 @@
+.primary {
```

Comment:
Consider putting these artifacts from Material Theme Builder into a separate folder, just so it's clear that's what they're from.

For example `example/HMI/app/css/material-theme/`?